File: 2980-check-if-bitwise-or-has-trailing-zeros/2980-check-if-bitwise-or-has-trailing-zeros.cpp
class Solution {
public:
    bool hasTrailingZeros(vector<int>& nums) {
        int count=0;
        
        
        for(int i=0;i<nums.size();i++){
            if(!(nums[i]&1))
                count++;
        }
        
        return (count>=2)?true:false;
        
    }
};

File: 2980-check-if-bitwise-or-has-trailing-zeros/2980-check-if-bitwise-or-has-trailing-zeros.py
class Solution:
    def hasTrailingZeros(self, nums: List[int]) -> bool:
        count = 0
    
        for num in nums:
            if not num & 1:
                count += 1
    
        return True if count >= 2 else False
    
        
        
        
        
        